custom/icds_reports/static/js/directives/map-or-sector-view/map-or-sector-view.directive.js

```diff
@@ -82,11 +82,13 @@ function MapOrSectorController($scope, $compile, $location, storageService, loca
     function navigateToLocation(locName) {
         locationsService.getLocationByNameAndParent(locName, location_id).then(function (locations) {
             var location = locations[0];
-            $location.search('location_name', location.name);
-            $location.search('location_id', location.location_id);
-            storageService.setKey('search', $location.search());
-            if (location.location_type_name === 'awc') {
-                $location.path(navigationService.getAWCTabFromPagePath($location.path()));
+            if (location) {
```

Comment:
this is actually a bugfix on mobile and web. it was previously raising js errors (which amount to the same behavior)